de_templates/ingest/Ingest_SourceType_SourceName_DQ/data_factory/pipelines/ARM_IngestTemplate.json.jinja

```diff
@@ -0,0 +1,267 @@
+                                                    "type": "Expression"
+                                                }
+                                            }
+                                        }
+                                    ]
+                                },
+                                {
+                                    "name": "Generate_Ingest_Query",
+                                    "type": "ExecutePipeline",
+                                    "dependsOn": [],
+                                    "userProperties": [],
+                                    "typeProperties": {
+                                        "pipeline": {
+                                            "referenceName": "Generate_Ingest_Query",
+                                            "type": "PipelineReference"
+                                        },
+                                        "waitOnCompletion": true,
+                                        "parameters": {
+                                            "ingest_entity_config": {
+                                                "value": "@item()",
+                                                "type": "Expression"
+                                            },
+                                            "window_start": {
+                                                "value": "@pipeline().parameters.window_start",
+                                                "type": "Expression"
+                                            },
+                                            "window_end": {
+                                                "value": "@pipeline().parameters.window_end",
+                                                "type": "Expression"
+                                            }
+                                        }
+                                    }
+                                }
+                            ]
+                        }
+                    },
+                    {
+                        "name": "Data_Quality",
+                        "type": "DatabricksSparkPython",
+                        "dependsOn": [
+                            {
+                                "activity": "For_Each_Ingest_Entity",
+                                "dependencyConditions": [
+                                    "Succeeded"
+                                ]
+                            }
+                        ],
+                        "policy": {
+                            "timeout": "0.12:00:00",
+                            "retry": 0,
+                            "retryIntervalInSeconds": 30,
+                            "secureOutput": false,
+                            "secureInput": false
+                        },
+                        "userProperties": [],
+                        "typeProperties": {
+                            "pythonFile": "dbfs:/FileStore/scripts/ingest_dq.py",
```

Comment:
We should probably upload to "dbfs:/FileStore/scripts/{{ pipeline_name }}/ingest_dq.py" or decide on some other approach to distinguish these scripts.

Reply:
Agreed 👍

Reply:
User story created, I will close this thread: https://amido-dev.visualstudio.com/Amido-Stacks/_workitems/edit/6534